packages/run-protocol/test/test-gov-collateral.js

```diff
@@ -74,15 +57,26 @@ const makeTestContext = async () => {
   const install = (src, dest) =>
     bundleCache.load(src, dest).then(b => E(zoe).install(b));
   const installation = {
-    liquidate: install(contractRoots.liquidate, 'liquidateMinimum'),
-    VaultFactory: install(contractRoots.VaultFactory, 'VaultFactory'),
-    amm: install(contractRoots.amm, 'amm'),
-    reserve: install(contractRoots.reserve, 'reserve'),
     mintHolder: install(contractRoots.mintHolder, 'mintHolder'),
     voting: install(contractRoots.voting, 'voting'),
```

Comment:
It looks like `voting` should go away from here for similar reasons.

Maybe `mintHolder` stays because it gets installed at chain bootstrap?

Reply:
Yes, that's right.  I didn't get back to removing `voting` after I added the `committee-invite-core.js`.